python/pyspark/sql/functions.py

```diff
@@ -67,9 +67,15 @@ def _():
     _.__doc__ = 'Window function: ' + doc
     return _
 
+_lit_doc = """
+    Creates a :class:`Column` of literal value.
 
+    >>> df.withColumn('height', lit(5)).withColumn('spark_user', lit(True)).collect()
+    [Row(age=2, name=u'Alice', height=5, spark_user=True),
+     Row(age=5, name=u'Bob', height=5, spark_user=True)]
```

Comment:
Here too, I think we can just do something like `df.select(hight, lit(5)).collect()` or `show()` to avoid `ignore_unicode_prefix`.